paasta_tools/cli/cmds/local_run.py

```diff
@@ -168,7 +168,7 @@ def run_healthcheck_on_container(
         healthcheck_result = perform_cmd_healthcheck(
             docker_client, container_id, healthcheck_data, timeout
         )
-    elif healthcheck_mode == "http" or healthcheck_mode == "https":
+    elif healthcheck_mode in ["http", "https", "http2"]:
```

Comment:
non-blocking suggestion: might be nice to have a constant for `HTTP_MODES = {"http", "https", "http2"}` since we have this list in multiple places